### PR TITLE
Topic 2 /Backend : *.py files translation

### PR DESCRIPTION
**Pull request by imenattatra**
## What type of PR is this? 
Related to the translation project : https://github.com/metr-systems/backlog/issues/3476

- [x] Feature - follow up

## Description

Things you need to know: 
- We use flask-babel for the translation, as mentioned [in the previous PR ](https://github.com/metr-systems/redash/pull/19)
- We set the translation to be conform to the original one , so that we translate it correctly in `redash/authentication/account.py`
- Some tests were having issues passing even if we are mocking the `flask_babel._` (`gettext`) for tests to always return the same value, so we had to use `force_locale("en") in them
- We tries to only translate what seems to be **client-viewed** texts and not **developper-views** texts, so I just ignored folders like `cli` and `destinations` and `logger` messages
`

## How is this tested?
- [x] Unit tests (pytest)

## What is next? (by order)
-> Translate the html files only from [templates folder](https://github.com/metr-systems/redash/tree/metr-main/redash/templates) in backend (soming soon)
-> Add flag feature for choosing the language (en/de)
-> Do more manual tests on staging and deploy 
-> Finish the rest of the translation

Redash spricht bald Deutsch !

These changes are currently deployed to staging

### CODE REVIEW

**Review by helenalebreton — APPROVED** [review]

File: redash/authentication/account.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def send_verify_email(user, org):
     context = {"user": user, "verify_url": verify_link_for_user(user)}
     html_content = render_template("emails/verify.html", **context)
     text_content = render_template("emails/verify.txt", **context)
-    subject = "{}, please verify your email address".format(user.name)
+    subject = _("{}, please verify your email address").format(user.name)
```
Comment (thiagogds) [inline redash/authentication/account.py:49]: This works fine? String substitution is sometimes tricky

**Review by thiagogds — APPROVED** [review]
Nice! There were way more text that I was expecting in the backend haha

File: redash/authentication/account.py (hunk 1 above)
Comment (imenattatra) [reply redash/authentication/account.py:49]: yes it does, I tried the version here in the PR for inviting myself on staging (not exactly this string substitution but the invitation one ) and it worked
Screenshot 2024-12-10 at 12 52 19

Comment (imenattatra) [reply redash/authentication/account.py:49]: the "Hi" was fixed in the other PR https://github.com/metr-systems/redash/pull/23

Comment (imenattatra) [reply redash/authentication/account.py:49]: thanks

**Pull request merged**